web/cypress/integration/plugin/create_and_delete_plugin.spec.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Delete Plugin List', () => {
+  const domSelectors = {
+    tableCell: '.ant-table-cell',
+    empty: '.ant-empty-normal',
+  };
+
+  beforeEach(() => {
+    cy.login();
+
+    cy.fixture('plugin-list.json').as('cases');
```

Comment:
why not use https://github.com/guoqqqi/apisix-dashboard/blob/fix-globalPlugin/web/cypress/fixtures/plugin-dataset.json and choose part of cases?